config-templates/azure/azuredisk-csi-driver/1.18.0/custom-parameters.json

```diff
@@ -20,7 +20,7 @@
         "mutable": "true"
     },
     {
-        "description": "Your Azure Active Directory Client ID. You can find your Client ID in the Azure portal or by running the 'az ad sp list --all' command.",
+        "description": "Your Azure Active Directory Client ID. You can find your Client ID in the Azure portal or by running the `az identity list` command.",
```

Comment:
This command is not correct. More specific command would be 
`az ad sp list --display-name appDisplayName`

Reply:
Same will be applicable in 1.23.0 as well

Reply:
Same will be applicable in 1.4.0 as well

Reply:
Fixed